### DISCUSSION TITLE
Steam Deck (/Proton)-friendly client?

### DISCUSSION THREAD

**Discussion by JpTiger** [Phalcode/gamevault-app#275] [General]
Hi there, I've been looking for a good way to host my own games on my NAS and manage/play them on my Steam Deck, including rips of my old PlayStation 2 game discs to play on PCSX2. Anyone here had any experience setting up something similar?

**Comment by Alfagun74** [comment]
The Dev-Team doesn't have a Steam Deck to test, but I've heard some people want to try something like this. Unfortunately, though, I've never heard of any success stories. If you manage it somehow, make sure to share it with us!

**Reply by JpTiger** [reply]
I haven't, but this person seems to have done it: https://www.reddit.com/r/SteamDeck/comments/17tskp2/comment/kalz7ry/?context=3

**Reply by rafaelvieiras** [reply]
Maybe just a Linux/Proton support is the that's enough to work on a Steam Deck.
